### DISCUSSION TITLE
how to Set id attribute on video tag?

### DISCUSSION THREAD

**Discussion by san-123** [GrapesJS/grapesjs#3739] [Q&A] [answered]
Hi @artf,

```
editor.BlockManager.add("html/video", {
            label: `<div>
                      <img src="../../../Images/SiteBuilder/video.svg"/>
                      <div class="my-label-block">Video</div>
                    </div>`,
            category: 'UI Controls',
            tab: 'UI Controls',
            id: 'UI Controls',
            content: `<video src="../../../Images/test.mp4" id="TestVideo"  type="video/mp4" >`,
        });
```

When drag and drop video tag with id attribute,  but Id attribute added automatically to div tag. see below
ex: 
"
```
 <div id="TestVideo" src="">
<video src=src="Images/test.mp4" type="video/mp4"></video>
</div>

```
How to add id attribute to video tag?

   

Please guide me.

**Comment by artf** [comment, marked as answer]
@san-123 it just doesn't make sense to have multiple elements with the same ID, try to rethink what you're doing...
https://stackoverflow.com/questions/5611963/can-multiple-different-html-elements-have-the-same-id-if-theyre-different-eleme

**Reply by san-123** [reply]
Hi @artf ,


Yes correct, but in video tag doest not create Id attribute.

Pl let me know.1

**Reply by artf** [reply]
This is how the video is rendered in the canvas and you should NEVER care about that. But again, do no use ids in your blocks.
